sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/MinedOpinion.java

```diff
@@ -3,43 +3,56 @@
 
 package com.azure.ai.textanalytics.models;
 
+import com.azure.ai.textanalytics.implementation.MinedOpinionPropertiesHelper;
 import com.azure.core.util.IterableStream;
 
 /**
  * The {@link MinedOpinion} model.
  */
 public final class MinedOpinion {
```

Comment:
I see, you kept the name of the object... I am not crazy about it.
What f you rename it to `SentenceOpinion`? or something that implies this is the opinion of a sentence

Reply:
@iscai-msft  how do you think about this name?